README.md

```diff
@@ -1,3 +1,167 @@
+2. Open a terminal and navigate to the newly checked out directory.
+3. Run `yarn install`, this will resolve all dependencies for you and install them in `node_modules`
+4. - Generate an RSA key-pair, if you have installed `openssl` and it's on your PATH, you can simply run `yarn generate-keypair`, which will create the files `privkey.pem` and `pubkey.pem` in the current directory.
+   - On productive systems you should move them into a "save directory" and (at least on UNIX-like systems) restrict read access of the private key to the user that's going to run the server and root.
+5. - Create a file `config.json` which contains all information the server needs to start.
+   - Alternatively those configuration options can be passed via CLI directly or as environment variables, check the [nconf docs](https://github.com/indexzero/nconf#example) for more information.
+   - The config files' Layout will be explained below.
+6. Create a Database in postgres to be used by the Server.
+   
+#### config.json
+```json
+{
+  "port": 7654,
+  "database": {
+    "username": "postgres",
+    "password": "",
+    "host": "localhost",
+    "port": 5432,
+    "database": "dicedb"
+  },
+  "email": {
+    "smtp": {
+      "host": "smtp.provider.com",
+      "port": 587,
+      "auth": {
+        "user": "your.email@provider.com",
+        "pass": "super secret password no one will ever guess"
+      }
+    },
+    "display": {
+      "sender": "\"Display Name\" <your.email@provider.com>",
+      "server": {
+        "protocol": "http",
+        "host": "localhost",
+        "port": 7654,
+        "baseurl": ""
+      }
+    }
+  },
+  "keys": {
+    "private": "./privkey.pem",
+    "public": "./pubkey.pem"
+  }
+}
+```
+- `port`: The port node.js will listen on. Required.
+- `database`: Details about the database connection. Required.
+   - `username`: Username to authenticate with the Database. Default: `postgres`.
+   - `password`: Password to authenticate with the Database. Default ` `.
+   - `host`: Hostname to connect to the Database. Default `localhost`.
+   - `port`: Port to connect to the Database. Default `5432`.
+   - `database`: Name of the Database to use, this should be the Database you created in the one-time setup. Default `dicedb`.
+- `email`: Settings that are used by the EmailManager. Required.
+   - `smtp`: Nodemailer SMTP Configuration, this object will be passed directly to Nodemailer without any further processing. Check the [Nodemailer Docs](https://nodemailer.com/smtp/#general-options). If you need a service to test emails locally you can use [Ethereal](https://ethereal.email/), a dummy email service that doesn't actually sends emails but simulates a fully-featured SMTP server.
+   - `display`: Display settings how the server will refer to itself in emails.
+      - `sender`: The Entry for the `From:` field in the email, the actual email should be the correct one, otherwise the emails will likely land in SPAM Folders. Required.
```

Comment:
```suggestion
      - `sender`: The Entry for the `From:` field in the email. The actual email should be the correct one, otherwise the emails will likely land in SPAM Folders. Required.
```